sql/report_queries/overdue_items/overdue_items.sql

```diff
@@ -0,0 +1,65 @@
+/* Item is overdue if number of days overdue (number of days between
+ * today and due date) is greater than 0.
+ * 
+ * Note: would like to include number of notices sent, but notices
+ * data are not yet available in LDP
+ * 
+ * FILTERS: 
+ * number of days a loan is overdue, 
+ * location name, and patron group name 
+ * if needed, e.g. Borrow Direct
+ * */
+
+WITH parameters AS (
+    SELECT
+        /* replace the placeholder number with the number of days overdue that is needed for this report */
+        '2000'::integer AS days_overdue_filter, -- doesn't work if empty
+        /* Fill in a location name, OR leave blank for all locations */
+        ''::varchar AS current_item_permanent_location_filter, --Online, Annex, Main Library
+        ''::varchar AS current_item_temporary_location_filter, --Online, Annex, Main Library
+        ''::varchar AS current_item_effective_location_filter, --Online, Annex, Main Library 
+        /*Choose a patron group name or leave blank*/
+        ''AS patron_group_filter --can use for Borrow Direct or ILL
```

Comment:
I tried this with ::varchar AS and it worked.

Reply:
Thanks so much! I think this is fixed now.